lib/notify.ex

```diff
@@ -1,4 +1,16 @@
 defmodule Notify do
+  @moduledoc """
+  Context module that handles the sending of actual notifications 
+  """
+  alias Notify.{Email, Mailer}
+
+  @doc false
+  @spec perform(map()) :: Bamboo.Email.t() | term()
+  def perform(%{approved: true} = pending_change), do: pending_change |> Email.change_approved() |> Mailer.deliver_now()
```

Comment:
We're not in the habit of one-lining pipelines so for the sake of consistency, could we use `do ... end` for our function and multiline the pipeline like we've done elsewhere in this PR?

Thank you 😁

Reply:
I considered that and thought meh! he will say this is wasting space

Reply:
In general I favor readability over saving whitespace, especially with community projects where I think it makes it easier for folks to quick see what's going on. Less is missed. 😁